echo_ext/middleware_ext/access_log.go

```diff
@@ -0,0 +1,74 @@
+package middleware_ext
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"strconv"
+	"time"
+)
+
+var (
+	_defaultLoggerAccessor = func(ctx echo.Context, startTime time.Time, endTime time.Time) {
+		logger := ctx.Logger()
+		request := ctx.Request()
+		response := ctx.Response()
+		requestID := request.Header.Get(echo.HeaderXRequestID)
+		if requestID == "" {
+			requestID = response.Header().Get(echo.HeaderXRequestID)
+		}
+		contentLength := request.Header.Get(echo.HeaderContentLength)
+		if contentLength == "" {
+			contentLength = "0"
+		}
+		logger.Infof("%s - %s %-7s %s %3d - %s %s %13v %s %s",
+			request.RemoteAddr,
+			request.Host,
+			request.Method,
+			request.RequestURI,
+			response.Status,
+			contentLength,
+			strconv.FormatInt(response.Size, 10),
+			endTime.Sub(startTime).String(),
+			request.Referer(),
+			request.UserAgent(),
+		)
+
+	}
+)
+
+type LoggerConfig struct {
```

Comment:
```suggestion
type AccessLogger struct {
```
应该是一个logger， 不是一个config吧

Reply:
这里的config的意思是对AccessLogger的一个配置项 LoggerConfig的Accessor可以从外部传输进去